packages/@sanity/cli/src/commands/init/initCommand.ts

```diff
@@ -3,6 +3,7 @@ import {detectFrameworkRecord, LocalFileSystemDetector} from '@vercel/fs-detecto
 
 import initPlugin from '../../actions/init-plugin/initPlugin'
 import initProject from '../../actions/init-project/initProject'
+import {type PackageManager} from '../../packageManager/packageManagerChoice'
 import {type CliCommandDefinition} from '../../types'
 
```

Comment:
```suggestion
import {ALLOWED_PACKAGE_MANAGERS, type PackageManager} from '../../packageManager/packageManagerChoice'
import {type CliCommandDefinition} from '../../types'

const allowedPms = ALLOWED_PACKAGE_MANAGERS.map(pm => `"${pm}"`).join(' | ')
```

Then use `${allowedPms}` in the help string, maybe?

Would help keep them in line.

Reply:
good point, I can make the change